Tor_KillTor_code.js

```diff
@@ -14,8 +14,8 @@ PROCESS_STANDART_OUTPUT = _result()
 sleep(1000)!
 native("filesystem", "removefile", RANDOM_FILE)
 PID = PROCESS_STANDART_OUTPUT.split('pid=')[1].split(' ')[0]
-RANDOM_FILE = "temp_" + rand() + ".bat"
```

Comment:
?

Reply:
It's bugging when creating it again. So adding a prefix for the filename for killing is solving the issue.

Reply:
Maybe It has a problem with the BAS. Have some problems with the **rand()** function. Maybe not working well for the twice calling. That's why I added the **Kill** prefix.

Reply:
Also adding **cmd /U** is making bat file cmd with UNICODE options. So It can successfully kill the task.